_docs/usecases/kubernetes/index.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: guide
+title: StorageOS Docs - Install applications on Kubernetes
+anchor: usecases
+module: usecases/kubernetes/examples
+---
+
+# Kubernetes
+
+StorageOS can be used to provide permanent storage for other applications
+running in [Kubernetes](https://kubernetes.io) or other Orchestrators that are
+derived from Kubernetes such as [OpenShift](https://openshift.com) or
+[Rancher](https://rancher.com). This is useful for running stateful
+applications, such as databases or CI/CD applications, under the control of
+Kubernetes as Kubernetes can make scheduling decisions withoutthe application
+data being lost.
+
+What we have outlined in the cookbooks below are some quick deployments of
+stateful applications into a Kubernetes cluster. These examples are not
+production ready but have been provided to give you some insight into how to
+use StorageOS with stateful applications.
+
+The examples we have provided use [Stateful sets](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/)
```

Comment:
We should be consistent with capitalisation and spelling of StatefulSet (use this version everywhere).

Reply:
(except in filenames which can stay lowercase)

Reply:
Done. I have also fixed everywhere else I have called it at Stateful set instead of StatefulSet (both lower and upper case)